framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/OpenShiftScenario.java

```diff
@@ -100,8 +102,8 @@ public final void deploy() {
         logger.info("Launch instances log collector on project {}", projectName);
         initLogCollectors();
 
-        logger.info("Creating generally used secret from " + OpenShiftTemplate.SECRET.getTemplateUrl().toString());
-        project.processTemplateAndCreateResources(OpenShiftTemplate.SECRET.getTemplateUrl(), Collections.singletonMap(OpenShiftConstants.SECRET_NAME, OpenShiftConstants.getKieApplicationSecretName()));
+        deploySecretConfig();
+        deploySecretAppUser();
```

Comment:
As you suggest, I named them as:
- deploySecretConfig, because it was the configuration related to the app
- deploySecretAppUser, because it was the configuration for the user credentials

Let me know what suits better for you and I will change it :)